randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -1924,53 +1912,37 @@ Extra - asset_id: collision_camera_025
 > Freezer Center; Heals? False
   * Layers: default
   > Door to Map Station
-      All of the following:
-          Any of the following:
-              Gravity Suit or Before Dairon - Power Switch 2
-              Heat/Cold Runs (Intermediate) and Cold Damage ≥ 150
-          Any of the following:
-              Space Jump or Speed Booster or Simple IBJ
-              All of the following:
-                  Wall Jump (Beginner)
-                  Flash Shift or Use Spin Boost
-              Morph Ball and Single-wall Wall Jump (Intermediate)
+      Any of the following:
+          Space Jump or Speed Booster or Simple IBJ
+          All of the following:
+              Wall Jump (Beginner)
+              Flash Shift or Use Spin Boost
+          Morph Ball and Single-wall Wall Jump (Intermediate)
   > Door to Power Switch 2
       Any of the following:
-          Gravity Suit
```

Comment:
Gravity Suit is still a valid option even with the power out since you can wall jump out of the water.